src/api/auth.rs

```diff
@@ -26,14 +26,20 @@ pub mod auth {
         }
     }
 
+    /// @param steam_id64 - The 64bit SteamId.
     /// @param timeoutSeconds - The number of seconds to wait for the ticket to be validated. Default value is 10 seconds.
     #[napi]
-    pub async fn get_session_ticket(timeout_seconds: Option<u32>) -> Result<Ticket, Error> {
+    pub async fn get_session_ticket_with_steam_id(
+        steam_id64: BigInt,
+        timeout_seconds: Option<u32>,
+    ) -> Result<Ticket, Error> {
         let client = crate::client::get_client();
         let (tx, rx) = oneshot::channel();
         let mut tx = Some(tx);
 
-        let (ticket_handle, ticket) = client.user().authentication_session_ticket();
+        let (ticket_handle, ticket) = client
+            .user()
+            .authentication_session_ticket_with_steam_id(SteamId::from_raw(steam_id64.get_u64().1));
```

Comment:
Does it makes sense to get a session ticket for another steam_id different from the one who is playing?

Reply:
Asking because I really don't know the answer, never used this on my games

Reply:
I have made a new commit and added comments of `authentication_session_ticket_with_steam_id`.